doc/services/portability/posix/option_groups/index.rst

```diff
@@ -909,18 +909,22 @@ _POSIX_TIMEOUTS
 _XOPEN_STREAMS
 ++++++++++++++
 
+With the exception of ``ioctl()``, functions in the ``_XOPEN_STREAMS`` option group are not
+supported in Zephyr at this time. Unsupported functions in this option group will fail with
+``ENOSYS`` :ref:`†<posix_undefined_behaviour>`.
```

Comment:
I think this is really confusing wording to say "... are not supported in Zephyr ..." in this paragraph only to follow with a table full of yesses.

Maybe the fix is to have the "Supported" column in the tables renamed to "Available?" or "Implemented" ?

Reply:
I think I will change the wording in the paragraph to "implement(ed)". It's probably more accurate.

However,  an implementation can still technically support a call (in that a conformant application can compile and link).